api/test/docker/manager-api-conf.yaml

```diff
@@ -36,7 +36,9 @@ authentication:
                              # if it's default value, when `manager api` start, it will generate a random string to replace it.
   expire_time: 3600          # jwt token expire time, in second
   users:                     # yamllint enable rule:comments-indentation
-    - username: admin        # username and password for login `manager api`
-      password: admin
-    - username: user
-      password: user
+    - type: local
+      username: admin   # username and password for login `manager api`
+      password: $2a$10$NjjlIhuj4O6foZhWw96H0uXZQl0na8qg.BSbf/ASwurchFhUW0fzC
```

Comment:
Why we should use the password's hash?

Reply:
Because the configuration files themselves are stored locally, rather than other centralized configuration management software such as etcd, these configuration files are scattered in the file systems of different machines, which are easy to leak and cause harm.
Using a simple bcrypt hash algorithm with time cost to process the password can ensure that even if the configuration file is illegally obtained, the password text will not be leaked.

If you think this part of the function is unnecessary, I can modify it back.

Reply:
#### Update
I have removed the password hash.